src/liveBlock.ts

```diff
@@ -31,13 +60,38 @@ const parse = (docParser: DocParser) => (block: Block): LiveBlock =>
         body: parseElements(docParser)(block.elements),
     })
 
-const parseLiveBlocks = (docParser: DocParser) => (blocks: Block[]): LiveBlock[] =>
-    blocks.map(parse(docParser));
+const parseMany = (blocks: Block[]): (docParser: DocParser) => LiveBlock[] =>
+    docParser => blocks.map(parse(docParser));
+
+const moreRecentThan = (since: Date) => (capiDate: CapiDateTime | undefined): boolean =>
+    maybeCapiDate(capiDate).fmap(date => date > since).withDefault(false);
+
+const filterBlocks = (filterFunc: (block: Block) => boolean) => (content: Content): Block[] =>
+    content.blocks?.body?.filter(filterFunc) ?? [];
+
+const blocksSince =
+    (getDate: (block: Block) => CapiDateTime | undefined) =>
+    (since: number): (content: Content) => (docParser: DocParser) => LiveBlock[] =>
+{
+    const isRecent = compose(moreRecentThan(new Date(since)), getDate);
+
+    return compose(parseMany, filterBlocks(isRecent));
+}
+
+const newBlocksSince = blocksSince(block => block.firstPublishedDate);
+const updatedBlocksSince = blocksSince(block => block.lastModifiedDate);
+
+const recentBlocks = (num: number) => (content: Content): (docParser: DocParser) => LiveBlock[] =>
```

Comment:
Will we also need a `blocksAround` when accessing key events?

Reply:
Yup, this is just the standard polling endpoint to start. There will be some more to implement to handle the other user interactions we support nowadays.